picturae_importer.py

```diff
@@ -367,7 +367,11 @@ def populate_taxon(self):
         """
         self.gen_spec_id = None
         self.taxon_list = []
-        self.taxon_id = self.sql_csv_tools.taxon_get(name=self.full_name)
+        if self.is_hybrid is False:
```

Comment:
taxon_get now has a different protocol for hybrids to better handle hybrids, where order is not standardized